src/Symfony/Component/Security/Http/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ CHANGELOG
 
  * `UserValueResolver` no longer implements `ArgumentValueResolverInterface`
  * Deprecate calling the constructor of `DefaultLoginRateLimiter` with an empty secret
+ * [BC BREAK] Remove experimental `InvalidSignatureException` and `MissingClaimException` classes from the `AccessToken\Oidc\Exception` namespace
```

Comment:
This BC break is not worth it IMO. you could deprecate this 2 exceptions instead and remove them in 7.0